### ISSUE TITLE
Colonists not eating.

### ISSUE THREAD

**Issue by ChaosTherum** [simon-82/ModVarietyPack#141]
Yes I know that this is not a public release yet but I figured it would be worth mentioning that since commit [2e28598](https://github.com/simon-82/ModVarietyPack/commit/2e28598e1bf6dd0d6ed167d0a4ac044a991729b6) Colonists have refused to eat or cook most growable products unless they can find from what I can tell shimmershrooms, meat, or milk there could be other things but these are just what I have noticed When they mix something growable such as potatoes it throws an exception when a colonist picks it up but otherwise works as expected though for example the colonists can not make a meal from berries, potatoes, and rice. Reverting to the previous commit removes the problem.

**Comment by simon-82** [comment]
thanks for reporting, but yes. There is no release yet. So it's kinda pointless tbh. 
We would have spotted this surely before releasing a new version.

Well, since you reported it, I might as well fix it now. 
Fix is uploaded in dev branch. 
Tell me if everything works now

**Comment by ChaosTherum** [comment]
Thanks I tried to fix it myself first I would love to be able to contribute and since I couldn't do it myself figured the next best things was to report it and help you narrow down a commit. Out of curiosity was it Items_Resource_Organic.xml That was about as far as I got in narrowing it down.

Edit: just checked the commit and I thought it might be the nutrition values but I was unsure if those were individually assigned or assigned by a parent object.
Also just curious if you have a list of goals or something maybe I could tinker around and see if I could implement something.

Edit 2: So plants seems to be working perfectly thanks for the quick response to the issue.
